### ISSUE TITLE
feat: create example screens with accessible and inaccessible examples

### ISSUE THREAD

**Issue by nbaddam** [nbaddam/throw#2]
create basic screens for demo purposes with common accessibility issues. the examples should be able to toggle between accessible and inaccessible examples as well.

all UI should have adequate contrast
